github.py

```diff
@@ -0,0 +1,271 @@
+# To-do list:
+# timed digests
+# listing/sorting/linking issues/pull requests
+# issue submission directly from Discord
+
+import os
+import os.path
+import aiohttp
+import asyncio
+import discord
+from datetime import datetime, timedelta
+from copy import copy
+from discord.ext import commands
+from cogs.utils.dataIO import dataIO
+
+
+class GitHub:
+    """Accesses GitHub."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.settingPath = "data/github/settings.json"
+        self.settings = dataIO.load_json(self.settingPath)
+        # async functions can't be called without being in another async function
+        # so _log_in and _wait_for_issue are added to the bot's event loop
+        self.bot.loop.create_task(self._update_server_list())
+        self.bot.loop.create_task(self._log_in())
+        self.bot.loop.create_task(self._wait_for_updates())
+
+    def _format_issue(self, issue, repo):
+        """Formats a GitHub issue nicely so it can be printed in Discord. issue is the dict returned by a GitHub
+        API request for a given Issue."""
+        string = "New issue on `{}` by {}: **{}**\n{}"
+        fullstring = string.format(repo, issue["user"]["login"], issue["title"], issue["html_url"])
+        return fullstring
+
+    async def _update_server_list(self):
```

Comment:
this doesn't need to be async

also, why are you loading all of the bot's servers into the settings?